### ISSUE TITLE
USWDS - Bug: Combo Box accessibility test needs to include details about default selection identification

### ISSUE THREAD

**Issue by CindySchaffner** [uswds/uswds-site#3264]
### Describe the bug

I was reviewing the page on testing Combo Boxes. I did not see any tests that include combo boxes that have a default selection preset. As a native screen reader user, this is very important to ensure any default selections are identified as selected when i tab and hear the element label. For example:  "Fruit Type combo box collapsed Bananas selected" 
As a 508 auditor for USCIS, this is one of the common findings we have with combo boxes. It forces the AT user to expand the drop down list, which again may or may not convey the default selection so they must select it themselves to be sure. At this point they still may not hear a status update confirming selection update.  
Often times devs forget to apply ARIA descendant attributes to these elements.  https://teams.microsoft.com/l/message/19:meeting_ZjFmYWRlODEtNDE1Ny00NWMzLWExODctNzlkODI5NjRlMjQ2@thread.v2/1757441109856?context=%7B%22contextType%22%3A%22chat%22%7D 

### Steps to reproduce the bug

Here is a link to the USWDS page where I did not see any test mentioning default selection identification: https://designsystem.digital.gov/components/combo-box/accessibility-tests/
Cannot share latest instance since it is an internal gov application.  But example flow would be as follows:
1. User tabs to combo box labeled  "Fruit Type", role is "combo box", and status is collapsed.
2. Visually tester can see "bananas are preselected by default, however this is not conveyed programmatically
3. User expands list and focus lands on the selected item Bananas. However, user still never hears 'selected'
4. User hits enter to select item, however status of selection is never updated or confirmed 

### Expected Behavior

1. . User tabs to combo box labeled  "Fruit Type", role is "combo box", and status is collapsed. Additionally, bananas is identified as selected
2. no other steps needed to identify defult selection status. 

A new internal DRUID library provides a clear example: https://pages.git.uscis.dhs.gov/DIDIT/druid/?path=/docs/components-select--design 

### Related code

https://designsystem.digital.gov/components/combo-box/accessibility-tests/

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by chandracarney** [comment]
In June, we discussed that this needs further testing and triage with [#5905](https://github.com/uswds/uswds/issues/5905). @jonathanbobel did a little digging again recently, and has been exploring solutions.

**Comment by chandracarney** [comment]
Thank you for reporting this, @CindySchaffner! We've added a test to our accessibility tests page, which should be up next time we push the site.
